src/index.html

```diff
@@ -2,16 +2,189 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"></form>
+    <fieldset class="fieldset-form">
+      <legend>Personal information</legend>
+
+      <div class="inputs">
+        <label for="surname-label">Surname:</label>
+        <input
+        type="text"
+        name="Surname"
+        id="surname-label"
+        >
```

Comment:
Fix code style. Ckec other places
```suggestion
        <input
          type="text"
          name="Surname"
          id="surname-label"
        >
```